docs/_docs/2_4_configurations.md

```diff
@@ -510,3 +510,30 @@ Property: `hoodie.memory.compaction.fraction` <br/>
 #### withWriteStatusFailureFraction(failureFraction = 0.1) {#withWriteStatusFailureFraction}
 Property: `hoodie.memory.writestatus.failure.fraction` <br/>
 <span style="color:grey">This property controls what fraction of the failed record, exceptions we report back to driver</span>
+
+### Write commit callback configs
+Controls callback behavior on write commit. Exception will be thrown if user enabled the callback service and errors occurred during the process of callback. Currently support http callback only, kafka implementation will be supported in the near future. 
+[withCallbackConfig](#withCallbackConfig) (HoodieWriteCommitCallbackConfig) <br/>
+<span style="color:grey">Callback related configs</span>
+
+#### HoodieWriteCommitHttpCallback
+
+##### writeCommitCallbackOn(callbackOn = false) {#writeCommitCallbackOn} 
+Property: `hoodie.write.commit.callback.on` <br/>
+<span style="color:grey">Turn callback on/off. off by default.</span>
+
+##### withCallbackClass(callbackClass) {#withCallbackClass} 
+Property: `hoodie.write.commit.callback.class` <br/>
+<span style="color:grey">Full path of user-defined callback class and must be a subclass of HoodieWriteCommitCallback class, org.apache.hudi.callback.impl.HoodieWriteCommitHttpCallback by default</span>
```

Comment:
move into Write commit callback configs section would be better?

Reply:
> move into Write commit callback configs section would be better?

yes, more clear